README.md

```diff
@@ -59,6 +59,13 @@ On a rooted Google OnHub, the command needs to be run with the "chronos" user. I
     # su - chronos
     Password:
     $ curl -Ls git.io/vddgY | bash
+    
+Compatablity
+------------
+
+If you I Crouton installed, Do I get support too?
+
+We will try to help, but we may not be able to. We will not give any extreme effort to provide support for Crouton (e.g. Removing or modifying packages in Chromebrew)
```

Comment:
This should go in the FAQ and not README.md.

Reply:
It is in the FAQ